_build/templates/default/package.json

```diff
@@ -1,25 +1,25 @@
 {
   "name": "revolution-theme-default",
   "title": "MODX Revolution",
-  "version": "2.3.0",
+  "version": "2.7.0",
```

Comment:
3.0.0?

Reply:
Indeed. First version was prepared for 2.7